docs-en/chapter_data_structure/classification_of_data_structure.md

```diff
@@ -20,29 +20,28 @@ Non-linear data structures can be further divided into tree structures and netwo
 
 ## Physical Structure: Contiguous and Dispersed
 
-**When an algorithm program runs, the data being processed is mainly stored in memory**. The following figure shows a computer memory stick, each black block containing a memory space. We can imagine memory as a huge Excel spreadsheet, where each cell can store a certain amount of data.
+**During the execution of an algorithm, the data being processed is stored in memory**. The figure below shows a computer memory stick where each black square is a physical memory space. We can imagine memory as a vast Excel spreadsheet in which each cell can store data of a specific size.
```

Comment:
This is just one suggestion to enhance the sentence's flow, but it's not necessary. Feel free to consider it as an optional improvement.

Suggestion:
We can think of memory as a vast Excel spreadsheet, with each cell capable of storing a certain amount of data.